src/app/routes/broker-ui/catalog-page/catalog-page/mapping/broker-catalog-mapper.ts

```diff
@@ -23,7 +23,9 @@ export class BrokerCatalogMapper {
       ...offer,
       asset: this.assetPropertyMapper.buildAsset({
         connectorEndpoint: offer.connectorEndpoint,
-        properties: offer.properties,
+        uiAsset: this.assetPropertyMapper.buildUiAssetfromprops(
+          offer.properties,
+        ),
```

Comment:
maybe simplify this to `{assetId: offer.properties['asset:prop:id']}`, so we can start deleting the old logic